Source/DiabloUI/title.cpp

```diff
@@ -13,19 +14,21 @@ std::vector<std::unique_ptr<UiItemBase>> vecTitleScreen;
 void TitleLoad()
 {
 	if (gbIsHellfire) {
+		// This is a 2.4 MiB PCX file without transparency (4.6 MiB as an SDL surface).

```

Comment:
Use cl2?

Reply:
We don't have a CL2 loader, only a CEL one, which won't help with the size here.
We can render PCX directly in a follow-up PR (see https://github.com/diasurgical/devilutionX/issues/4301)

Reply:
> We don't have a CL2 loader, only a CEL one, which won't help with the size here.

I'm not sure I follow. At some point we didn't have a CEL loader either.

> We can render PCX directly in a follow-up PR (see https://github.com/diasurgical/devilutionX/issues/4301)

Should we try and load CL2 as PCX to simplify things in the rendere?

Reply:
In PCX, colors <= 0xBF can be encoded as a single byte, while other colors always require a run-length prefix, which means applying a palette transform requires reallocation.

For PCX, we don't have a renderer yet but we'd only need a simple one that doesn't handle clipping, as we'd only use it for the backgrounds.

Loading PCX as CL2 is extra work/allocation and is not easier than implementing a non-clipping PCX renderer.

Reply:
> This is out-of-scope for this PR though.

Of cause not, I was just wondering about possible future steps :)